road_following_model_pth: best_steering.pth
img_recognition_model_pth: best3-2021-02-23-15-31-49.pth
use_cuda: true

road_preprocess_mean: [0.485, 0.456, 0.406]   # image parameter for road_following
road_preprocess_stdev: [0.229, 0.224, 0.225]  # image parameter for road_following

img_preprocess_mean: [0.485, 0.456, 0.406]    # image parameter for img_recognition
img_preprocess_stdev: [0.229, 0.224, 0.225]   # image parameter for img_recognition

